.travis.yml

```diff
@@ -13,6 +13,9 @@ env:
     - CONDA_NPY=19  CONDA_PY=34  
     - CONDA_NPY=110  CONDA_PY=35  
     - CONDA_NPY=19  CONDA_PY=35  
+  global:
```

Comment:
Nope. But it was not there: https://github.com/conda-forge/basemap-feedstock/blob/2290f22eefe7c04abd361ca309bd82d08af950c3/appveyor.yml